typespec-tests/Generate.ps1

```diff
@@ -43,13 +43,18 @@ $generateScript = {
   } elseif ($tspFile -match "resiliency[\\/]srv-driven[\\/]main.tsp") {
     # enable advanced versioning for resiliency test
     $tspOptions += " --option ""@azure-tools/typespec-java.advanced-versioning=true"""
```

Comment:
For comment https://github.com/Azure/autorest.java/pull/2722#discussion_r1580502052

We could just add `api-version=all` to this case.

Reply:
Added.